lib/bundle/utils.rb

```diff
@@ -24,6 +24,18 @@ def self.cask_installed?
     @@cask ||= Kernel.system("brew cask --version &>/dev/null")
   end
 
+  def self.installed_formulae
+    @@installed_formulae ||= `brew list -1`.split("\n")
+  end
+
+  def self.outdated_formulae
+    @@outdated_formulae ||= `brew outdated --quiet`.split("\n")
+  end
+
+  def self.pinned_formulae
+    @@pinned_formulae ||= `brew list --pinned`.split("\n")
+  end
```

Comment:
It seems that these methods wouldn't be used outside `BrewInstaller`. So maybe put them inside `BrewInstaller`?